import_chembl.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+import gzip
+import pandas
+import tempfile
+import urllib
+
+def get_arguments():
+    parser = argparse.ArgumentParser(description='Download ChEMBL entries and convert them to input for preprocessing')
+    DEFAULT_URI = 'ftp://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/latest/chembl_22_chemreps.txt.gz'
```

Comment:
please pull this out to between the imports and the `def get_arguments` declaration following the example in the other scripts